CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Added popout diff visualizer for table properties like Attributes and Tags ([#834])
 * Updated Theme to use Studio colors ([#838])
 * Added experimental setting for Auto Connect in playtests ([#840])
+* Fixed http error handling so Rojo can be used in Github Codespaces ([#847])
```

Comment:
I'm not sure I think this is the best entry (it feels like working with GitHub Codespaces is a side effect) but we can revisit it before the next release if I still care.

Reply:
While sorta a side effect, it was the reason and impact for this PR.